invenio_communities/views/api.py

```diff
@@ -0,0 +1,177 @@
+                Content-Length: 334
+                Content-Type: application/json
+                [
+                    {
+                        "id": "comm1"
+                    },
+                    {
+                        "id": "comm2"
+                    }
+                ]
+            :resheader Content-Type: application/json
+            :statuscode 200: no error
+        """
+        communities = Community.query.all()
+        result = []
+        for community in communities:
+            result.append({
+                'id': community.id
+            })
+        return {'json': result}
+
+
+class CommunityDetailsResource(ContentNegotiatedMethodView):
+    """"Community details resource."""
+
+    def __init__(self, serializers=None, *args, **kwargs):
+        """Constructor."""
+        super(CommunityDetailsResource, self).__init__(
+            serializers,
+            *args,
+            **kwargs
+        )
+
+    def get(self, community_id):
+        """Get the details of the specified community.
+
+        .. http:get:: /communities/(string:id)
+            Returns a JSON dictionary with the details of the specified
+            community.
+            **Request**:
+            .. sourcecode:: http
+                GET /communities/communities/comm1 HTTP/1.1
+                Accept: application/json
+                Content-Type: application/json
+                Host: localhost:5000
+            :reqheader Content-Type: application/json
+            :query string id: ID of an specific community to get more
+                              information.
+            **Response**:
+            .. sourcecode:: http
+                HTTP/1.0 200 OK
+                Content-Length: 334
+                Content-Type: application/json
+                {
+                    "id_user": 1,
+                    "description": "",
+                    "title": "",
+                    "created": "2016-04-05T14:56:37.051462",
+                    "id": "comm1",
+                    "page": "",
+                    "curation_policy": ""
+                }
+
+            :resheader Content-Type: application/json
+            :statuscode 200: no error
+            :statuscode 404: page not found
+        """
+        community = Community.query.get(community_id)
+        if community:
+            return {
+                'json': {
+                    'id': community.id,
+                    'id_user': community.id_user,
+                    'title': community.title,
+                    'description': community.description,
+                    'page': community.page,
+                    'curation_policy': community.curation_policy,
+                    'created': community.created.isoformat(),
+                }
+            }
+        abort(404)
+
+
+serializers = {'application/json': json_serializer}
+
+
+communities_view = CommunitiesResource.as_view(
+    'communities_api',
+    serializers=serializers
```

Comment:
I think ``default_media_type`` should also be set in order to make sure we return ``application/json`` in case no ``Accept`` header was provided. Ditto for below.